owncloudApp/src/main/res/layout/account_action.xml

```diff
@@ -16,33 +16,40 @@
   You should have received a copy of the GNU General Public License
   along with this program.  If not, see <http://www.gnu.org/licenses/>.
 -->
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:id="@+id/linearLayout"
```

Comment:
It is not a linear layout anymore... Maybe the ID also deserves some changes in its name 😁

Reply:
Totally right!